src/test/java/marquez/client/BackendsTest.java

```diff
@@ -1,68 +1,95 @@
 package marquez.client;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNullPointerException;
 
 import com.google.common.collect.ImmutableMap;
 import java.io.IOException;
 import org.junit.Test;
 
 public class BackendsTest {
+  private static final String API_KEY = "PuRx8GT3huSXlheDIRUK1YUatGpLVEuL";
+
+  @Test
+  public void testHttp_throwsOnNull() {
+    assertThatNullPointerException().isThrownBy(() -> Backends.newHttpBackend(null));
+    assertThatNullPointerException()
+        .isThrownBy(() -> Backends.newHttpBackend(Backends.DEFAULT_BASE_URL, null));
+  }
 
   @Test
   public void testHttp() {
-    ImmutableMap<String, String> env =
+    final ImmutableMap<String, String> env =
         ImmutableMap.<String, String>builder()
             .put("MARQUEZ_BACKEND", "HTTP")
             .put("MARQUEZ_URL", "https://localhost:8080")
             .build();
-    Backend backend = Backends.newBackendFromEnv(env);
-    assertTrue(backend.getClass().getName(), backend instanceof HttpBackend);
+    final Backend backend = Backends.newBackendFromEnv(env);
+    assertThat(backend).isInstanceOf(Backend.class);
+    assertThat(((HttpBackend) backend).http.apiKey).isNull();
+  }
+
+  @Test
+  public void testHttpWithApiKey() {
+    final ImmutableMap<String, String> env =
+        ImmutableMap.<String, String>builder()
+            .put("MARQUEZ_BACKEND", "HTTP")
+            .put("MARQUEZ_URL", "https://localhost:8080")
+            .put("MARQUEZ_API_KEY", API_KEY)
+            .build();
+    final Backend backend = Backends.newBackendFromEnv(env);
+    assertThat(backend).isInstanceOf(Backend.class);
+    assertThat(((HttpBackend) backend).http.apiKey).isEqualTo(API_KEY);
+  }
+
+  @Test
+  public void testFile_throwsOnNull() {
+    assertThatNullPointerException().isThrownBy(() -> Backends.newFileBackend(null));
```

Comment:
I think this test is not really necessary, but feel free to keep it.

Reply:
I'll keep it. It increases our test coverage (_slightly_).